ui/packages/statement/src/translations/zh-CN.yaml

```diff
@@ -1,5 +1,8 @@
 statement:
   nav_title: SQL 语句分析
+  pages:
+    overview: SQL 语句总览
```

Comment:
For entry pages, there should be no titles

Reply:
Removed:

![image](https://user-images.githubusercontent.com/1284531/79345514-cb72a700-7f63-11ea-9ef9-7f52f93ab73a.png)